views/helpers.go

```diff
@@ -16,37 +25,47 @@ type (
 		Callback string
 		// User is the stored logged-in user
 		User user.User
+		// Version is the version that is running
+		Version string
```

Comment:
Why does this need to be a separate type?

Reply:
But why can't it just be a `string` - what's the value of it having its own type alias?

Reply:
It is just a string?

Reply:
So it is - the code folding on GitHub made it look like it was inside a `type (` and I missed the opening struct. (Part of why I don't like grouping together types in `type ()` blocks, but I recognise that's a losing battle these days :sob:)